### DISCUSSION TITLE
Can AtataNavigator help us get rid of AtataContext.Current?

### DISCUSSION THREAD

**Discussion by Piedone** [atata-framework/atata#502] [Q&A] [answered]
In our `async`-heavy test suite we're doing this pattern currently:

```c#
// myContext is an AtataContext carried in our custom context object.
AtataContext.Current = myContext;
Go.To<TPage>();
```

Is the new `AtataNavigator` designed to make this easier with the following?

```c#
myContext.Go.To<TPage>();
```

Thank you!

**Comment by YevgeniyShunevych** [comment, marked as answer]
Right @Piedone, it should be this way. But unfortunately, it doesn't work right now because of #504. Let me make a fix release of Atata v1.11.1 tomorrow.

**Reply by Piedone** [reply]
Thanks for the quick reply, Yevgeniy! I had such a hunch because I didn't see the context being populated anywhere in the code.

**Reply by YevgeniyShunevych** [reply]
v1.11.1 is released and `Go.To` functionality should hopefully work well for your case.

**Reply by Piedone** [reply]
Thanks, works indeed.
